src/usr/local/www/system.php

```diff
@@ -221,7 +221,7 @@ function is_timezone($elt) {
 		if ($_POST[$dnsgwname] && ($_POST[$dnsgwname] <> "none")) {
 			foreach ($direct_networks_list as $direct_network) {
 				if (ip_in_subnet($_POST[$dnsname], $direct_network)) {
-					$input_errors[] = sprintf(gettext("A gateway can not be assigned to DNS '%s' server which is on a directly connected network."), $_POST[$dnsname]);
+					$input_errors[] = sprintf(gettext("A gateway cannot be specified for %s because that IP is part of a directly connected subnet %s. To use that nameserver, change its Gateway to `none`."), $_POST[$dnsname], $direct_network);
```

Comment:
Change "IP" to "IP address" or "address" ([ref](https://www.netgate.com/docs/pfsense/references/style-guide.html#what-to-avoid))

Reply:
Done.